model/clusters_mgmt/v1/aws_sts_policies_type.model

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an sts policies for rosa cluster
+class AWSSTSPolicies {
```

Comment:
I don't see any place in the API that manages only one of these objects. That means that they don't have their own identity, therefore the `id` and `href` attributes will be meaningless. It would then be better to use here `struct` instead of `class`.

I'd also say that as this represents a single policy, it should be named `AWSTSPolicy`.